.github/CODEOWNERS

```diff
@@ -0,0 +1,52 @@
+###################################################################################################
+# Overview
+#
+# Pattern used to match files follows most of the same rules as used in gitignore files. Order is
+# important; the last matching pattern takes precendence.
+#
+# For more info see:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+###################################################################################################
+
+
+###################################################################################################
+# Global (@keycloak/maintainers)
+###################################################################################################
+
+*                                                           @keycloak/maintainers
+
+###################################################################################################
+# Testing (@keycloak/testing-maintainers)
+###################################################################################################
+
+/testsuite/                                                 @keycloak/testing-maintainers
```

Comment:
@stianst Can we avoid creating a new group for `testing-maintainers` and change this line to the following or we don't want this?
```suggestion
/testsuite/                                                 @keycloak/cloud-native-maintainers @keycloak/store-maintainers @keycloak/ui-maintainers
```

Reply:
Absolutely, clearly I didn't ready the docs for CODEOWNERS file well enough.

Made the owners core, cloud-native, and store though. UI have their own Cypress testsuites.

Reply:
Thanks @stianst @mhajas!

Doesn't UI team also own the `base-ui` Arquillian testsuite? I know they are rewriting it to Cypress but for the time being it might make sense to also add `ui-maintainers` here. Or maybe just add `/testsuite/integration-arquillian/tests/other/base-ui` to `ui-maintainers`?

Reply:
Added